packages/react-components/src/components/EditBox.tsx

```diff
@@ -39,8 +39,13 @@ export const EditBox = (props: EditBoxProps): JSX.Element => {
   const { onCancel, onSubmit, initialValue, strings } = props;
   const [textValue, setTextValue] = useState<string>(initialValue);
   const [textValueOverflow, setTextValueOverflow] = useState(false);
+  const editTextFieldRef = React.useRef<ITextField>(null);
   const theme = useTheme();
 
+  useEffect(() => {
+    editTextFieldRef.current?.focus();
+  }, []);
```

Comment:
Doesn't lint complain about dependencies here? `editTextFieldRef.current` should be in the dependency array.

Reply:
no complain, having an empty array will go through this only when component mount, and that's what we want to be able to have the cursor on the TextTield when EditBox opens